src/js/routes/index.js

```diff
@@ -49,12 +55,20 @@ function getApplicationRoutes() {
   routes = [
     {
       type: Route,
+      component: Index,
       children: [
         {
           type: Route,
           id: 'index',
-          component: Index,
-          children: routes
+          children: routes,
+          component: PassThroughComponent
+        },
+        {
+          // Please make sure this is THE LAST route and is not inside of id: 'index'
```

Comment:
// Please make sure this is THE LAST route and is not inside of id: 'index'
// as this is a glob route and matching stops here
// making all the routes going after this point unreachable